modules/installation-osp-deploying-bare-metal-preexisting.adoc

```diff
@@ -0,0 +1,65 @@
+[id="installation-osp-deploying-bare-metal-preexisting_{context}"]
+= Deploying a cluster that uses bare-metal compute machines on user-provisioned infrastructure
+// = Preparing to deploy a cluster that uses bare-metal compute machines on user-provisioned infrastructure
+//FIXME title is essentially the assembly--mod it
+You must modify the `install-config.yaml` file prior to deploying a cluster that uses bare-metal compute machines
+on an existing, user-provisioned {rh-openstack-first} network. In this configuration, control plane machines are
+deployed on VMs.
+
+.Prerequisites
+
+* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible by using the {rh-openstack} Compute API.
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* {rh-openstack} is running on network infrastructure that you provisioned.
+
+* An {rh-openstack} subnet is provisioned.
+
+* The {rh-openstack} network supports both VM and bare metal server attachment.
+
+* You created an `install-config.yaml` file as part of the {product-title} installation process.
+
+.Procedure
+
+. In the `install-config.yaml` file, edit the flavors for `compute.platform.openstack.type` and `controlPlane.platform.openstack.type`:
++
+.An example bare metal `install-config.yaml` file
+[source,yaml]
+----
+                controlPlane:
+                   platform:
+                     openstack:
+                       type: my-vm-control-plane-flavor <1>
+                 ...
+
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: my-bare-metal-flavor <2>
+                   replicas: 3
+                 ...
+                 platform:
+                   openstack:
+                     machinesSubnet: 25495f6a-8eae-43ff-ad7b-77ba57ed0a04 <3>
+----
+<1> Change this value to the VM flavor that is used for control plane machines.
+<2> Change this value to the bare metal flavor that is used for compute machines.
+<3> Change this value to the UUID of the {rh-openstack} subnet that is already provisioned.
+
+Use the updated `install-config.yaml` file to complete the installation process.
+The compute machines that are created during deployment use the flavor that you
+added to the file.
+
+// Which is essentially:
+// - Run the openshift installer:
+
+//       ./openshift-install create cluster --log-level debug
+
+// - wait for the installer to complete.
+
+//     If the installer times out waiting for the bare-metal workers to complete booting,
```

Comment:
Is this a UX bug?

Reply:
Yes. For which unfortunately we don't have a clear solution at the moment.